src/sa_web/static/js/views/app-view.js

```diff
@@ -505,11 +523,16 @@ var Shareabouts = Shareabouts || {};
       this.setBodyClass();
     },
     newPlace: function() {
+      console.log("appView.newPlace");
       // Called by the router
-      this.collection.add({});
+      this.collection.places.add({});
     },
-    // TODO: Refactor this into 'viewPlace'
-    viewLandmark: function(model, options) {
+
+    // REFACTOR
+    // add a datasetSlug parameter here, which corresponds to the id property from config.yml
+    viewLandmark: function(datasetSlug, model, options) {
```

Comment:
I would recommend not refactoring this `viewLandmark` method just yet, and instead focus on refactoring `viewPlace` first. Later, we would like to refactor `viewLandmark` and `viewPlace` for DRYness, but let's keep things explicit and not worry too much about it yet.